src/Public/Core.ps1

```diff
@@ -150,6 +152,26 @@ function Start-PodeServer {
         $PodeContext = $null
         $ShowDoneMessage = $true
 
+        # check if podeWatchdog is configured
+        if ($PodeWatchdog) {
+            if (!($null -eq $PodeWatchdog.DisableTermination -or
+                    $null -eq $PodeWatchdog.Quiet -or
+                    $null -eq $PodeWatchdog.PipeName -or
+                    $null -eq $PodeWatchdog.Interval )
+            ) {
```

Comment:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWl2bTNncXVzcDQwbTNybGFjNHp1dWxlaDAxcXZsZWJ1YzlvMmRvMSZlcD12MV9naWZzX3NlYXJjaCZjdD1n/3owzW5c1tPq63MPmWk/giphy.gif)

This may be simpler to understand.
```suggestion
            if ($null -ne $PodeWatchdog.DisableTermination -or
                    $null -ne $PodeWatchdog.Quiet -or
                    $null -ne $PodeWatchdog.PipeName -or
                    $null -ne $PodeWatchdog.Interval 
            ) {
```